amethyst_input/src/button.rs

```diff
@@ -17,6 +17,9 @@ pub enum Button {
     /// Mouse buttons
     Mouse(#[serde(with = "LocalMouseButton")] MouseButton),
 
+    /// Mouse wheel
+    MouseWheel,
```

Comment:
I think we need to add this as axis to the `InputHandler`. Also, an example would be nice ;)

Reply:
I've thought about handling that as Axis too, but we have to consider that we have to split this into 2 Axis that are handled by this Event, with delta_x = horizontal scrolling, delta_y = vertical scrolling, and i'm currently not having a good idea how to integrate the emulated axis with that.

I'm thinking about an example implementation too, may its possible to extend the arc_ball_camera example with distance-control referring to the mousewheel, but this was just my first idea.